doc/api/test.md

```diff
@@ -920,6 +920,61 @@ test('runs timers as setTime passes ticks', (context) => {
 });
 ```
 
+## Snapshot testing
+
+> Stability: 1.0 - Early development
+
+Snapshot tests allow arbitrary values to be serialized into string values and
+compared against a set of known good values. The known good values are known as
+snapshots, and are stored in a snapshot file. Snapshot files are managed by the
+test runner, but are designed to be human readable to aid in debugging. Snapshot
+files should be checked into source control along with your test files. In order
+to enable snapshot testing, Node.js must be started with the
```

Comment:
```suggestion
to enable snapshot testing, Node.js should be started with the
```
not sure about all these documentation fixes, I am not a native English speaker so feel free to resolve these without changing.

Reply:
"Must" implies that something has to be done. "Should" implies that it is probably best to do it, but it is not a requirement. In this case, I think "must" is correct because the APIs will not be available without the CLI flag.

Reply:
https://datatracker.ietf.org/doc/html/rfc2119